assets/js/modules/analytics/datastore/__fixtures__/index.js

```diff
@@ -42,12 +42,6 @@ export { default as dashboardBounceRateWidgetData } from './dashboard-bounce-rat
 export { default as dashboardBounceRateWidgetArgs } from './dashboard-bounce-rate-widget-args.json';
 export { default as dashboardGoalsWidgetData } from './dashboard-goals-widget-data.json';
 export { default as dashboardGoalsWidgetArgs } from './dashboard-goals-widget-args.json';
```

Comment:
I think we can remove a bunch more of these files now since they appear to be no longer used.

Reply:
Yeah, managed to remove the remaining reports data fixtures.